docs/source/advanced/cmake.rst

```diff
@@ -238,6 +238,11 @@ ALPAKA_ACC_GPU_HIP_ONLY_MODE
 
      Only back-ends using HIP can be enabled in this mode.
 
+ALPAKA_HIP_FAST_MATH
+  .. code-block::
+
+     Enable fast-math.
```

Comment:
Could you please add a warning box: `The default value is changed with alpaka 0.7.0 to "OFF"`

Later we need to add a note to the changelog into the breaking changes section too.

Reply:
We should discuss this today in the dev meeting.

Reply:
On the VC we decided to make this addition.